collections/_drafts/agent-forwarding.md

```diff
@@ -79,10 +79,11 @@ all tools requiring services from the agent.
 
 If the option `--enable-ssh-support` is used, the auto-start mechanism does not
 work because _ssh_ does not know about this mechanism. Instead, the environment
-variable `SSH_AUTH_SOCK` must be set to the `S.gpg-agent.ssh` socket in the GnuPG
-home directory. Further, `gpg-agent` must be started by either using a
-GnuPG command that implicitly starts `gpg-agent` or by using `gpgconf --launch
-gpg-agent` to explicitly start it without first having to use a GnuPG command.
+variable `SSH_AUTH_SOCK` must be set to the `S.gpg-agent.ssh` socket in the
+GnuPG home directory. Further, `gpg-agent` must be started by either using a
+GnuPG command that implicitly starts `gpg-agent` or by using
+`gpgconf --launch gpg-agent` to explicitly start it without first having to use
+a GnuPG command.
 
 `gpg-agent` is a daemon to manage secret (private) keys independently from any
```

Comment:
```suggestion
`gpg-agent` is a daemon to manage secret (private) keys independently from any
```

hats off to yan for writing up some good habits of subkey generation, that was my next plan on writing definitely going to be using that as a reference (still gotta make sure its right first, too haha):

🔗  https://github.com/diracdeltas/documentation/blob/master/gpg-usage.markdown#subkey-generation